src/components/Calendar/Calendar.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+import NextArrow from '../Icons/NextArrow';
+import PrevArrow from '../Icons/PrevArrow';
+import * as CalendarFunc from './utils';
```

Comment:
`CalendarFunc` is a bad name (also it should not be in camel case). Personally, I prefer explicit import in such cases. I mean `import { ... } from '...'`